wordpress-mobile/WordPress-Android
# Issue/13326 fix threat found redirection

Parent issue #13326

With the introduction of the Scan screen, we can introduce support for redirection to Scan screen from the notifications. Previously, the app would redirect the user to a webview.

Merge instructions:
1. Make sure [FluxC's PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1859) is merged first
2. Update FluxC tag
2. Remove "Not ready for merge label


To test:
Prerequisities:
- disable Scan feature flag
1. Go to Notifications
2. Click on a "Jetpack Scan detected a possible security threat" item
3. Click on "Site's Scan Results" span
4. Notice a webview is shown
---------------------------
Prerequisities:
- enable Scan feature flag
1. Go to Notifications
2. Click on a "Jetpack Scan detected a possible security threat" item
3. Click on "Site's Scan Results" span
4. Notice Scan screen is shown
5. Run the test again, but make sure that you have selected a different site (SiteB) in the app than the one (SiteA) on which the threat was found -> make sure the Scan screen is shown for SiteA


PR submission checklist:
